docs/developer-center/new-guides/quickstart.md

```diff
@@ -1,185 +1,966 @@
 
-Note: In case your data hasn't been geocoded before, you can do it thanks to our Location Data Services. Learn how to geocode your data reading the [geocoding guide]().
+```python
+from cartoframes.data.services import Geocoding
+
+starbucks_df, _ = Geocoding().geocode(starbucks_df, street='address')
+starbucks_df.head()
+```
 
-You can visualize your geodataframes using the [Map]() and [Layer]() classes. Check the [visualization guide]() to know all the visualization possibilities and check the [sources guide]() to know about which data sources are supported.
 
-```py
+<div>
+<style scoped>
+    .dataframe tbody tr th:only-of-type {
+        vertical-align: middle;
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+
+    .dataframe thead th {
+        text-align: right;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>name</th>
+      <th>address</th>
+      <th>revenue</th>
+      <th>gc_status_rel</th>
+      <th>carto_geocode_hash</th>
+      <th>geometry</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>1</th>
+      <td>Franklin Ave &amp; Eastern Pkwy</td>
+      <td>341 Eastern Pkwy,Brooklyn, NY 11238</td>
+      <td>1321040.772</td>
+      <td>0.91</td>
+      <td>9212e0e908d8c64d07c6a94827322397</td>
+      <td>POINT (-73.95901 40.67109)</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>607 Brighton Beach Ave</td>
+      <td>607 Brighton Beach Avenue,Brooklyn, NY 11235</td>
+      <td>1268080.418</td>
+      <td>0.97</td>
+      <td>b1bbfe2893914a350193969a682dc1f5</td>
+      <td>POINT (-73.96122 40.57796)</td>
+    </tr>
+    <tr>
+      <th>3</th>
+      <td>65th St &amp; 18th Ave</td>
+      <td>6423 18th Avenue,Brooklyn, NY 11204</td>
+      <td>1248133.699</td>
+      <td>0.95</td>
+      <td>e47cf7b16d6c9b53c63e86a0418add1d</td>
+      <td>POINT (-73.98976 40.61912)</td>
+    </tr>
+    <tr>
+      <th>4</th>
+      <td>Bay Ridge Pkwy &amp; 3rd Ave</td>
+      <td>7419 3rd Avenue,Brooklyn, NY 11209</td>
+      <td>1185702.676</td>
+      <td>0.95</td>
+      <td>2f21749c02f73116892eb3b6fd5d5738</td>
+      <td>POINT (-74.02744 40.63152)</td>
+    </tr>
+    <tr>
+      <th>5</th>
+      <td>Caesar's Bay Shopping Center</td>
+      <td>8973 Bay Parkway,Brooklyn, NY 11214</td>
+      <td>1148427.411</td>
+      <td>0.95</td>
+      <td>134c23973313802448365db6235783f9</td>
+      <td>POINT (-74.00098 40.59321)</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+Done! Now that the stores are geocoded, you will notice a new column called `geometry` has been added. This column stores the geographic location of each store and it's used to plot each location on the map.
+
+You can quickly visualize your geocoded dataframes using the Map and Layer classes. Check the [visualization guide]() to know all the visualization possibilities.
```

Comment:
replace: dataframes with dataframe

replace: "Check the [visualization guide]() to know all the visualization possibilities." with "Check out the [visualization guide]() to learn more about the visualization capabilities inside of CARTOframes."